packages/admin-stories/src/docs/components/ClearInputButton/stories/CustomClearIcon.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { ClearInputButton } from "@comet/admin";
+import { Box, Input, Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { Cut } from "../../../../../../admin-icons/lib";
+
+storiesOf("stories/components/Clear Input Button/Custom Clear Icon", module).add("Custom Clear Icon", () => {
+    return (
+        <Box display={"flex"} alignItems={"center"}>
+            <Box marginRight={15}>
+                <Typography variant={"body1"}>Input Field with Custom ClearInputButton Icon:</Typography>
+            </Box>
+            <Input endAdornment={<ClearInputButton icon={() => <Cut />} />} />
```

Comment:
What's the motivation behind using a function for the `icon` prop? Passing the `disabled` argument? Shouldn't this work out of box when the ClearInputButton is disabled?

Reply:
Thats a good question :) i am not sure i just wrote the docs and used it how it was defined by the person who developed it :)